blocks/footer/footer.js

```diff
@@ -105,6 +106,10 @@ function decorateDisclaimer(block) {
   defaultWrapper.appendChild(panelBody);
 }
 
+function stickyFooter() {
+  return div({ class: ['sticky-footer', 'turnstile-container'] });
```

Comment:
Are we using this dom-builder because it's safe & clean to write a DOM rather than a string, or is it  a known pattern that we are following across projects? 
I'm wondering if it's readable, especially for a bigger DOM, compared to just typing out as string literal directly.

Reply:
Looks good but never have a chance to try a big DOM

Reply:
Yes, using the dom-builder here. Let's see if we find it useful in this project. If not, I'll revert it.